src/main/java/bio/terra/workspace/app/controller/GlobalExceptionHandler.java

```diff
@@ -1,6 +1,6 @@
 package bio.terra.workspace.app.controller;
 
-import bio.terra.workspace.common.exception.ErrorReportException;
+import bio.terra.common.exception.ErrorReportException;
```

Comment:
Can we also change to subclassing the TCL's AbstractGlobalExceptionHandler? Doesn't have to be this PR, but a quick follow up if not would be good.
https://github.com/DataBiosphere/terra-common-lib/blob/develop/src/main/java/bio/terra/common/exception/AbstractGlobalExceptionHandler.java

Reply:
Yes, will follow up.